spec/acceptance/selenium_server_spec.rb

```diff
@@ -9,8 +9,7 @@
     # Using puppet_apply as a helper
     it 'should work with no errors' do
       pp = <<-EOS
-        include java
-        Class['java'] -> Class['selenium::server']
+        Class['windows::java'] -> Class['selenium::server']
```

Comment:
This will need to be made conditional so posix platforms aren't declaring `windows::java`.  Note that in this example `windows::java` isn't actually included in the manifest.